libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -83,6 +89,57 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("FS_VOLTSRC", 11, AP_BattMonitor_Params, _failsafe_voltage_source, BattMonitor_LowVoltageSource_Raw),
 
+    // @Param: LOW_VOLT
+    // @DisplayName: Battery low voltage
+    // @Description: Battery voltage to trigger a low battery failsafe. Set to 0 to disable. If the battery voltage drops below this voltage continuously for 10 seconds then the vehicle will switch to RTL mode.
+    // @Units: V
+    // @Increment: 0.1
+    // @User: Standard
+    AP_GROUPINFO("LOW_VOLT", 12, AP_BattMonitor_Params, _low_voltage, DEFAULT_LOW_BATTERY_VOLTAGE),
+
+    // @Param: LOW_MAH
+    // @DisplayName: Battery low capacity
```

Comment:
I'd be tempted by "Low battery capacity" or "Low battery capacity failsafe trigger level".

Reply:
I like the former more. Updated, I'm on the fence if the voltage should be arranged similarly. Does any GCS use this display name to sort a list? (IE does this change risk breaking parameter grouping for them?)

Reply:
Just checked with @meee1 it's grouped by the param name rather then the display name, so keeping the more natural version is fine. Should the voltage one be rearranged as well then? (IE `Low battery voltage`)

Reply:
Up to you, but IMO they should be consistent.

Reply:
Done.